tests/x11regressions/libreoffice/libreoffice_mainmenu_components.pm

```diff
@@ -74,53 +74,51 @@ sub run() {
 
     $self->open_mainmenu();
     assert_and_click 'mainmenu-office-impress';    #open impress
-    assert_screen 'ooimpress-select-a-template';
-    send_key "alt-f4";                             #close template window
+    send_key 'alt-f4' if check_screen 'ooimpress-select-a-template';    #if there is template window, close it
```

Comment:
Please don't do this `check_screen` with no timeout by default. You will waste 30 seconds here.

Better use a multi-tag `assert_screen` plus `match_has_tag`. Or, if you must, use `check_screen …, 0`